README.md

```diff
@@ -109,35 +153,131 @@ Now in a slurm job array the work will be performed (you can set the number of a
 
 ## Running on Grid
 
```

Comment:
As said above, it would be better to decouple the fractals example as grid-only and show how to run it locally, on slurm and grid. So the title should be about running a long running or advanced example (fractals)